packages/data-store/src/utils/contact/MappingUtils.ts

```diff
@@ -103,7 +110,7 @@ export const contactFrom = (contact: BaseContactEntity): Contact => {
 }
 
 export const isNaturalPerson = (contact: NonPersistedContact | BaseContactEntity): contact is NonPersistedNaturalPerson | NaturalPersonEntity =>
-  'firstName' in contact && 'lastName' in contact
+  'firstName' in contact && 'lastName' in contact && !('grade' in contact) && !('dateOfBirth' in contact)
```

Comment:
is this correct? this is old code right ? there is no grade and dateOfBirth

Reply:
Orphaned code, good catch. 
Removed.